src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,62 @@
+const { expensesService } = require('../services/expenses.service');
+const { usersService } = require('../services/users.service');
+
+class ExpensesController {
+  getAllExpenses = (req, res) => {
+    res.send(expensesService.getAll(req.query));
+  };
+  createExpense = (req, res) => {
+    const { userId } = req.body;
+    const user = usersService.getById(userId);
+
+    if (!user) {
+      res.sendStatus(400);
+
+      return;
+    }
+
+    return res.status(201).json(expensesService.create(req.body));
```

Comment:
```suggestion
try {
    const user = usersService.getById(userId);

    if (!user) {
      res.sendStatus(404);

      return;
    }

    return res.status(201).json(expensesService.create(req.body));
} catch (err) {
    res.statusCode = 500;
    res.send('Something went wrong);
}
```
Also, it would be better to wrap this bunch of code inside the try-catch block to handle errors related to the DB connection, etc. In this case, we should provide a client with a proper error code and avoid throwing an error object directly to the client.

Reply:
Also, try to implement these suggestions for all of your code